.github/workflows/qt6-build.yml

```diff
@@ -28,5 +28,10 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: "{{defaultContext}}:docker/qt6-build"
+          platforms: linux/amd64,linux/arm64
+          build-args: |
+            UBUNTU_VERSION=20.04
+            QT_MAJOR=6.5
+            QT_VERSION=6.5.6
           push: true
-          tags: mltframework/qt:6.4.3-ubuntu18.04
+          tags: mltframework/qt:6.5.6-ubuntu20.04
```

Comment:
Why did you change this version? Did you build this and verify that builds using the version of g++ in Ubuntu 20.04. The reason this says ubuntu:18.04 is because GitHub Actions times out building this, and I need to build it locally and push the image. So, this workflow is not really used anymore.

Reply:
`mltframework/qt:6.4.3-ubuntu18.04` is not available, but `mltframework/qt:6.4.3-ubuntu20.04` IS. The shotcut-build image for x86-64 MUST remain the way it is currently--on mltframework/qt:6.4.3-ubuntu20.04`. Remove all  changes to this file because it not used currently, and you must build the qt image yourself.